openhtf/io/output/output_to_file.py

```diff
@@ -0,0 +1,32 @@
+"""Output to file.
+
+Output modules inherit from OutputToFile and overload methods
+based on output file requirements.
+
+"""
+
+import os
+import tempfile
+
+
+class OutputToFile(object):
```

Comment:
I'd recommend putting this in `io/output/__init__.py`, it's kind of a "base output functionality", not an extension thereof, so deserves to be a first-class citizen, I think.